x-pack/test/send_search_to_background_integration/tests/apps/dashboard/async_search/sessions_in_space.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import { getSearchSessionIdByPanelProvider } from './get_search_session_id_by_panel';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+  const security = getService('security');
+  const PageObjects = getPageObjects([
+    'common',
+    'header',
+    'dashboard',
+    'visChart',
+    'security',
+    'timePicker',
+  ]);
+  const getSearchSessionIdByPanel = getSearchSessionIdByPanelProvider(getService);
+  const browser = getService('browser');
+  const sendToBackground = getService('sendToBackground');
+
+  describe('dashboard in space', () => {
+    describe('Send to background in space', () => {
+      before(async () => {
+        await esArchiver.load('dashboard/session_in_space');
+
+        await security.role.create('data_analyst', {
+          elasticsearch: {
+            indices: [{ names: ['logstash-*'], privileges: ['all'] }],
+          },
+          kibana: [
+            {
+              base: ['all'],
```

Comment:
This test is great!
I think we can further limit the privileges here to assert that our new sub-feature privilege is working correctly from end-to-end:

```suggestion
              feature: {
                dashboard: ['minimal_read', 'store_search_session'],
              },
```

Reply:
Lets hope this passes :crossed_fingers: